### ISSUE TITLE
Ajouter la possiblité de revenir au tri initial dans le tableau de l'observatoire

### ISSUE THREAD

**Issue by lucaa** [DISIC/wikidemarches#859]
Le tri implémenté pour #719 ne permet pas d'annuler un tri selectionné , il est uniquement possible de changer le tri vers un autre critère mais pas d'enlever le tri choisi.
La seule possiblité pour revenir au tri initial est de recharger le tableau.

Pour rappel, le tri par défaut est celui décrit dans  #636 .

**Comment by lucaa** [comment]
Il faut décider comment on veut implémenter cette fonctionnalité, du point de vue de l'opération des boutons de tri (voir captures dans https://github.com/DISIC/wikidemarches/issues/719#issuecomment-582552982 ): 
* afficher une option "revenir au tri par défaut" dans la liste d'options ou 
* afficher l'option de tri courante dans la liste d'options et donner la possibilité de cliquer une deuxième fois dessus pour la "deselectionner" et revenir à l'option initiale

**Comment by AnthonyBrunelli** [comment]
Pour mémoire, j'avais fait une proposition pour les filtres dans la fiche fermée #719 que je replace ici pour discussion.
![73254752-4a570f80-41bf-11ea-80e7-29d9546ec13c](https://user-images.githubusercontent.com/35567048/74015139-5cd80280-4990-11ea-8cf4-06899cbe05a9.png)

**Comment by lucaa** [comment]
@AnthonyBrunelli effectivement, j'avais fermé #719 sans répondre à cette proposition, partiellement parce-que ce changement n'est pas critique pour répondre au sujet de la tâche. 
Ainsi, je propose de mettre cette nouvelle proposition dans une tâche séparée, pour avoir une discussion dédiée à ce changement.

**Comment by lucaa** [comment]
@AnthonyBrunelli j'ai créé #862 pour ce changement d'affichage, je l'ai mis dans la colonne pour validation interne Dinum avant de commencer dessus.

**Comment by AnthonyBrunelli** [comment]
Bien ouej
